src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -587,18 +587,27 @@
 	outline-color: var(--vscode-focusBorder);
 }
 
-/* Overrides for the a11y buffer as we're hosting monaco within it */
-.xterm .xterm-accessible-buffer {
-	padding: 0 !important;
-	overflow: initial !important;
-	overflow-x: initial !important;
-}
-.xterm .xterm-accessible-buffer:focus-within {
-	top: 0 !important;
-	left: 0 !important;
-	pointer-events: all !important;
-}
-.xterm > .xterm-accessible-buffer {
+.accessible-buffer {
```

Comment:
We should prefix this with something as the name is too generic atm. `.monaco-workbench .terminal-accessible-buffer` is good